lib/rails_event_store/event_repository.rb

```diff
@@ -0,0 +1,11 @@
+module RailsEventStore
```

Comment:
Sure, the other way to do this is to have the dependency actually load the proper repository by setting the RailsEventStore adapter at run time.

Would that work for you?